Content.Shared/WebPlacer/WebPlacerComponent.cs

```diff
@@ -0,0 +1,79 @@
+using Content.Shared.Actions;
+using Content.Shared.Whitelist;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Shared.WebPlacer;
+
+/// <summary>
+/// Gives the entity (probably a spider) an ability to spawn entites (probably webs) around itself.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+[Access(typeof(SharedWebPlacerSystem))]
+public sealed partial class WebPlacerComponent : Component
+{
+    /// <summary>
+    /// Id of the entity getting spawned.
+    /// </summary>
+    [DataField(customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string WebPrototype = "SpiderWeb";
```

Comment:
It's generally best practice to use ProtoIds directly, now.

```suggestion
    [DataField]
    public ProtoId<EntityPrototype> WebPrototype = "SpiderWeb";
```